tools/klee/main.cpp

```diff
@@ -1118,12 +1122,17 @@ linkWithUclibc(StringRef libDir, std::string opt_suffix,
 }
 
 int main(int argc, char **argv, char **envp) {
-  atexit(llvm_shutdown);  // Call llvm_shutdown() on exit.
+  atexit(llvm_shutdown);  // Call llvm_shutdown() on exit
 
-#if LLVM_VERSION_CODE >= LLVM_VERSION(13, 0)
-  KCommandLine::HideOptions(llvm::cl::getGeneralCategory());
-#else
-  KCommandLine::HideOptions(llvm::cl::GeneralCategory);
+  std::set<llvm::cl::OptionCategory *> kleeCategories = {
+      &ChecksCat,      &DebugCat,    &ExtCallsCat, &ExprCat,   &LinkCat,
+      &MemoryCat,      &MergeCat,    &MiscCat,     &ModuleCat, &ReplayCat,
+      &SearchCat,      &SeedingCat,  &SolvingCat,  &StartCat,  &StatsCat,
+      &TerminationCat, &TestCaseCat, &TestGenCat};
+  KCommandLine::KeepOnlyCategories(kleeCategories);
+
+#if LLVM_VERSION_CODE >= LLVM_VERSION(11, 0)
+  KCommandLine::HideOptions(PollyCategory);
 #endif
```

Comment:
This change should now be unnecessary
```suggestion
```

Reply:
Yes, I still need to clean up the code, will do.   I was only expecting a review once the CI passes ;)